src/content/docs/data-apis/custom-data/aws-privatelink.mdx

```diff
@@ -500,6 +541,19 @@ These are the New Relic endpoint services available via AWS PrivateLink:
             `com.amazonaws.vpce.us-west-2.vpce-svc-006678e8c6f037152`
           </td>
         </tr>
+        <tr>
+          <td>
+
+          </td>
```

Comment:
@haus Is there supposed to be something in this cell?

Reply:
same comment as above. the cell is intentionally blank to imply a continuation of the product from the row immediately above. i copied that pattern from the us-east-2 table